python/paddle/fluid/layers/tensor.py

```diff
@@ -466,6 +466,14 @@ def assign(input, output=None):
           fluid.layers.assign(data, result1) # result1 = [[2.5, 2.5], [2.5, 2.5], [2.5, 2.5]]
           result2 = fluid.layers.assign(data)  # result2 = [[2.5, 2.5], [2.5, 2.5], [2.5, 2.5]]
           result3 = fluid.layers.assign(np.array([[2.5, 2.5], [2.5, 2.5], [2.5, 2.5]], dtype='float32')) # result3 = [[2.5, 2.5], [2.5, 2.5], [2.5, 2.5]]
+
+          i = fluid.layers.fill_constant(shape=[1], value=0, dtype='int64')
+          array = fluid.layers.array_write(x=data, i=i)
+          result_array1 = fluid.layers.create_array(dtype='float64')
+          fluid.layers.assign(array, result_array1)
+          result_array2 = fluid.layers.assign(array)
+          result4 = fluid.layers.array_read(array=result_array1, i=i) # result4 = [[2.5, 2.5], [2.5, 2.5], [2.5, 2.5]]
```

Comment:
result4, 5 is very ugly. (result 1, 2 above is fine since number is small).
Let's change "result4, 5" into "array_elem1, 2".

Reply:
Done.